code/modules/client/preference_setup/loadout/loadout_gloves.dm

```diff
@@ -92,14 +92,18 @@
 	ringtype["engagement ring"] = /obj/item/clothing/gloves/ring/engagement
 	ringtype["signet ring"] = /obj/item/clothing/gloves/ring/seal/signet
 	ringtype["masonic ring"] = /obj/item/clothing/gloves/ring/seal/mason
-	ringtype["ring, steel"] = /obj/item/clothing/gloves/ring/material/steel
+	ringtype["ring, glass"] = /obj/item/clothing/gloves/ring/material/glass
+	ringtype["ring, wood"] = /obj/item/clothing/gloves/ring/material/wood
+	ringtype["ring, plastic"] = /obj/item/clothing/gloves/ring/material/plastic
 	ringtype["ring, iron"] = /obj/item/clothing/gloves/ring/material/iron
+	ringtype["ring, bronze"] = /obj/item/clothing/gloves/ring/material/bronze
+	ringtype["ring, steel"] = /obj/item/clothing/gloves/ring/material/steel
+	ringtype["ring, copper"] = /obj/item/clothing/gloves/ring/material/copper
 	ringtype["ring, silver"] = /obj/item/clothing/gloves/ring/material/silver
 	ringtype["ring, gold"] = /obj/item/clothing/gloves/ring/material/gold
+	ringtype["ring, phoron"] = /obj/item/clothing/gloves/ring/material/phoron
```

Comment:
Because @MistakeNot4892 prompted me for further input and I actually remembered: Yeah if gaseous phoron gets into your clothes, it will cause toxloss until you take them off and wash them. In light of that, this seems a bit nonsensical